lib/DataProviderInterface.js

```diff
@@ -35,7 +35,7 @@
  * @method
  * @name DataProvider#fetchTransaction
  * @param {string} id
- * @returns {Promise<{ confirmations: number }|null>}
+ * @returns {Promise<RawTransaction|null>}
```

Comment:
Where did you define it?

Reply:
it's undefined yet. Should I use Object instead?

Reply:
Yeah, it's better than specify undefined type.